### ISSUE TITLE
Just to let you know, the library you excluded (Audio Interface Library) was open-sourced in 2000.

### ISSUE THREAD

**Issue by fpusoft** [2point21/lba1-classic#3]
http://ke5fx.com/
http://www.thegleam.com/ke5fx/misc/AIL2.ZIP AIL Version 2, the complete Open Source release of the Miles Sound System Version 2 for DOS

**Comment by xesf** [comment]
@fpusoft it has been added to the LBA2 repo: https://github.com/2point21/lba2-classic/tree/main/LIB386/ail

Not sure the contents are exactly the same.
@sviannay @gwen-gg Can you confirm they have the same contents? or do we need to push the LBA1 AIL version eventually?

**Comment by fpusoft** [comment]
> 
> 
> @fpusoft it has been added to the LBA2 repo: https://github.com/2point21/lba2-classic/tree/main/LIB386/ail
> 
> Not sure the contents are exactly the same. @sviannay @gwen-gg Can you confirm they have the same contents? or do we need to push the LBA1 AIL version eventually?

Contents in LBA2 aren't the same.
The AIL version released as open source is the same one used in LBA1.
Check the file "REL105.ZIP"

For the other AIL version for LBA2, maybe RAD Game Tools would agree to open source it since its over 20 years old.

ÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄ
IBM Audio Interface Library for 32-bit DOS (AIL/32)
Release 1.05 of 19-Oct-93
ÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄ

                             ÚÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄ¿
                             ³ List of files ³
                             ÀÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÙ

SAMPLE   AD  ........... Sample YM3812 (Ad Lib etc.) Global Timbre Library
SAMPLE   OPL ........... Sample OPL-3 (Ad Lib Gold etc.) Global Timbre Library
AIL32    ASM ........... Application Program Interface source file
DMASND32 ASM ........... Digital (sampled) sound driver source
XMIDI32  ASM ........... Extended MIDI (XMIDI) sequence interpreter source
DLLLOAD  C   ........... DLL loader for OS/2 32-bit Linear Executable modules
MIX32    C   ........... Sample program source
STP32    C   ........... Sample program source
VP32     C   ........... Sample program source
XP32     C   ........... Sample program source
AIL32    H   ........... C header file containing all function prototypes
DLL      H   ........... C header file for DLLLOAD.C functions
AIL32    INC ........... Miscellaneous assembly equates
MPU40132 INC ........... Roland MPU-401-compatible I/O primitives
MT3232   INC ........... Roland MT-32 synthesizer driver source
SBMIDI32 INC ........... Sound Blaster MIDI port I/O primitives
SPKR32   INC ........... Tandy/IBM internal speaker synthesizer driver source
YAMAHA32 INC ........... Yamaha YM3812 & YMF262/OPL3 synthesizer driver source
386      MAC ........... 32-bit DOS assembly macros
A32MP    MAK ........... Sample Microsoft makefile for MetaWare/Phar Lap build
A32WR    MAK ........... Sample Microsoft makefile for Watcom/Rational build
A32ZF    MAK ........... Sample Microsoft makefile for Zortech/Flashtek build
VPRICE   VOC ........... Sample Creative Labs .VOC (PCM digitized sound) file
DEMO     XMI ........... XMIDI file containing BACKGND, SHANTY, CHORAL.MID
SPKRDEMO XMI ........... XMIDI file for PC/Tandy internal speaker playback
DOS4GW   EXE ........... Rational Systems 32-bit DOS extender for Watcom C
MSMAKE   EXE ........... Microsoft MAKE program to process sample makefiles
READ     ME  ........... This file

Versions of the following files are present in the subdirectories
\WR, \MP, and \ZF.  These filesets are compiled for use with the 
Watcom/Rational, MetaWare/Phar Lap, and Zortech/Flashtek 
development environments, respectively.

A32ADLIB DLL ........... Standard Ad Lib XMIDI sound driver DLL
A32ALGDG DLL ........... Ad Lib Gold PCM sound driver DLL
A32ALGFM DLL ........... Ad Lib Gold XMIDI sound driver DLL
A32MT32  DLL ........... Roland MT-32 XMIDI sound driver DLL (MPU401 MIDI)
A32MT32S DLL ........... Roland MT-32 XMIDI sound driver DLL (SBlaster MIDI)
A32PASDG DLL ........... Pro Audio Spectrum PCM sound driver DLL
A32PASFM DLL ........... Pro Audio Spectrum XMIDI sound driver DLL
A32PASOP DLL ........... Pro Audio Spectrum Plus/16 XMIDI sound driver DLL
A32SBDG  DLL ........... Sound Blaster PCM sound driver DLL
A32SBFM  DLL ........... Sound Blaster XMIDI sound driver DLL
A32SBPDG DLL ........... Sound Blaster Pro PCM sound driver DLL
A32SP1FM DLL ........... Sound Blaster Pro (YM3812) XMIDI sound driver DLL
A32SP2FM DLL ........... Sound Blaster Pro (OPL-3) XMIDI sound driver DLL
A32SPKR  DLL ........... IBM PC internal speaker XMIDI sound driver DLL
A32TANDY DLL ........... Tandy 3-voice internal speaker XMIDI sound driver DLL
MIX32    EXE ........... Sample program: plays simultaneous XMIDI sequences 
STP32    EXE ........... Sample program: plays raw 22 kHz. stereo samples
VP32     EXE ........... Sample program: plays Creative Labs .VOC files
XP32     EXE ........... Sample program: plays XMIDI sequences
AIL32    OBJ ........... Linkable API (Process Services) object module

**Comment by sviannay** [comment]
Yes, I saw this was released in open-source (LBA1 version only), but after discussed, we decided to not include it as part of the LBA1 delivery as it can be retrieved from original one (I think it's better to not fork it). I hesitated to do so, but maybe we could just point to https://github.com/Tronix286/AIL2 (and/or from http://ke5fx.com/ you noticed, I didn't know this intersting link) or at least add a mention that an open-source version exists.

**Comment by Freso** [comment]
It might also work to add it as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules) pointing to https://github.com/Tronix286/AIL2 ? That way the code would still work as-is from a clone (with `submodule init` and `update`) but the code wouldn’t be duplicated/"forked" in this repository but rather still point to the upsteam.
